### ISSUE TITLE
Gradle-server doesnt start if /bin/sh doesnt exist

### ISSUE THREAD

**Issue by RandomModderJDK** [microsoft/vscode-gradle#1341]
Extension Name: vscode-gradle
Extension Version: v3.6.1
OS Version: Linux mint 22.04
VSCode version: v1.78

**Describe the bug**
If you do not have access to /bin/sh, because you have some other shell installed, it does not work.

**To Reproduce**
(You probably should not do this on debian or so)
1. Delete /bin/sh
2. Try using the extension
3. See that /bin/sh is not found

**Expected behavior**
Enviroments with no sh shell installed, should be able to change the shell.

**Output from "Gradle for Java"**
```
[info] [gradle-server] /bin/sh: /home/xxx/.vscode/extensions/richardwillis.vscode-gradle-3.6.1-universal/lib/gradle-server: No such file or directory 

[warn] [gradle-server] Gradle server stopped
```

**Does the bug still exist if you disable all other extensions?**
Yes

**Additional context**
My solution I thought of, is to just make the option described [here](https://github.com/microsoft/vscode-gradle/blob/653c096633864ea8c895e00587b76aa8eb0828dc/extension/src/server/GradleServer.ts#L56) more accessible to users. This may be specified in the settings.

**Comment by RandomModderJDK** [comment]
My temporary workaround was/is replacing the the "shell: true" in the index.js locally.
This works nicely, so no other cause.

**Comment by RandomModderJDK** [comment]
Should I do this, so this issue can be closed? It feels like a duplicate:

> Feature request: open a GitHub issue to allow configuring the shell used to launch the Gradle server (see code at
> 
> [vscode-gradle/extension/src/server/GradleServer.ts](https://github.com/microsoft/vscode-gradle/blob/653c096633864ea8c895e00587b76aa8eb0828dc/extension/src/server/GradleServer.ts#L56)
> 
> Line 56 in [653c096](https://github.com/microsoft/vscode-gradle/commit/653c096633864ea8c895e00587b76aa8eb0828dc)
>  shell: true, 
> ). A setting like gradle.server.shellPath could override the hard-coded /bin/sh.

**Comment by chagong** [comment]
@RandomModderJDK set shell: false could have some impact on other platform like win32. the command line here is a .bat in win32, and that could cause an error. Since your case is very corner case, we won't plan to make the change for now.

**Comment by RandomModderJDK** [comment]
Right. I mean it is a one time change and it shouldn't bother to many people to update this when needed. 
I don't require to always update so it's fine.
